docs/tutorials/intro.md

```diff
@@ -70,102 +72,72 @@ algokit init
 
-This should produce something similiar to the following in the VSCode terminal.
+## Run the Demo Application 🏃‍♂️
 
-```shell
-(playground-py3.11) User@Algo-User-MBP myalgokit project % "/Users/user/code/algokit/myalgokit pro
-ject/.venv/bin/python" "/Users/user/code/algokit/myalgokit project/playground/hello_world/demo.py"
-Deployed app in txid SBNIJYZUOXVPXDFJHAVDDCO5TQ4WZ7P37QOOQM6CSVSMIURETHNQ
-        App ID: 11
-        Address: 377KDIVHB7K2LFFJIPUPQFWJGVD36MAP4EI6Y2HAGRAW6JPK4MDUBC3YVY
+Once the `Algorand Python` based project is created, you can get started by building and deploying your first smart contract. There are a few possible ways to do so as demonstrated on the following video.
 
-Hello, Beaker
-```
+[![Watch the video](https://ipfs.algonode.xyz/ipfs/Qmc9mRaPoDyhUFmek4ETxVfKUKzUg9pf3Tss5xwkBGdQis)](https://ipfs.algonode.xyz/ipfs/Qma6gNqxsSFc9Jbh8kBTZyVLv5gqFj1xnrsjoeT6MAAwCw/)
 
-The App ID of of the deployed contract and its Algorand address is displayed, followed by the message returned from the smart contract call (`Hello, Beaker`).
+As shown above, click on clicking on `Run and Debug` and then `Build & Deploy contracts`. Alternatively, you can hit F5 (or whatever keyboard shortcut is set in VS Code for running the debugger). While you are viewing the `contract.py` file - it will:
 
-At this point you have deployed a simple contract to an Algorand network and called it successfully!
+1. Start LocalNet
+2. Build the smart contract. Notice how a folder named `artifacts` is created with the TEAL and JSON files for the smart contract.
+3. Deploy and call the `HelloWorld` smart contract (from `contract.py` file).
 
-Additionally, you can find the native TEAL smart contract code and the appropriate smart contract manifest JSON files have been output to the `artifacts` folder.
+After execution, the `App ID` of of the deployed contract and its Algorand address are displayed, followed by the message returned from the smart contract call (`Hello, world`).
 
-![AlgoKit Playground Demo](../imgs/algokitartifacts.png)
+At this point you have deployed a simple contract to a LocalNet Algorand network (running in your Docker environment) and called it successfully!
 
-Running the `build.py` python file will also generate these artifacts without deploying. These files can be used by tools like [Dappflow](https://dappflow.org/), [goal](https://developer.algorand.org/docs/clis/goal/goal/), etc. to deploy your smart contract to the various Algorand networks.
+Additionally, you can find the native TEAL smart contract code and the appropriate smart contract manifest JSON files have been output to the `artifacts` folder. Note that in this example, TEAL code is compiled using `PuyaPy` compiler, which is what `Algorand Python` relies on to represent your Python smart contract code in TEAL ([Transaction Execution Approval Language](https://developer.algorand.org/docs/get-details/dapps/avm/teal/)).
```

Comment:
```suggestion
Additionally, you can find the native TEAL smart contract code and the appropriate smart contract manifest JSON files have been output to the `artifacts` folder. Note that in this example, TEAL code is compiled using the `PuyaPy` compiler, which is what `Algorand Python` relies on to compile your Python smart contract code to TEAL ([Transaction Execution Approval Language](https://developer.algorand.org/docs/get-details/dapps/avm/teal/)).
```

Reply:
@neilcampbell remove excessive usage of word TEAL in favour of AVM bytecode